index.html

```diff
@@ -0,0 +1,190 @@
+                    </div> 
+                    </div>
+                    <div class="elements-short">
+                      <a href="#">SEE THE WHOLE SERVICES</a>
+                    </div>
+                    <div class="joincc">
+                      <a href="#">Join CC Global Summit 2022</a>
+                    </div>
+
+                 </article>
+                     
+                 <aside class="speakers-cont">
+                     <div class="speakers-cont-title">
+                      <h3>Featured Speakers</h3>
+                      <div class="bar"></div>
+                     </div>
+
+                    <div id="speakers" class="fs"></div>
+  
+                 </aside>
+             </section>
+            <footer class="footer-section">
+                    <div>
+                      <h6>Sponsors</h6>
+                       <div class="bar"></div>
+                    </div>
+                     <div class="footer-partners">
+                         <div class="partners">
+                           <a href=""><img src="./images/msf1.png" alt="Mozilla"></a>
+                         </div>
+                         <div class="partners-icon">
+                           <a href=""><img src="./images/mozila.svg" alt="Google"></a>
+                         </div>
+                         <div class="partners-icon">
+                             <a href=""><img src="./images/bb.png" alt="naver"></a>
+                         </div>
+                         <div class="partners">
+                              <a href=""><img src="./images/share-image.png" alt="tokopedia"></a>
+                         </div>
+                         <div class="partners-icon">
+                              <a href=""><img src="./images/airbnb.svg" alt="airbnb"></a>
+                         </div>
+                    </div>
+                    
+                    <div class="footer-container">
+                      <div class="container-child">
+                        <div class="footer-title">&copy; ICTNET AFRICA <br>DR. CONGO
+                        </div>
+                          <p class="footertext">
+                            Creative commons DR. Congo. Some rights reserve<br> <br>Lorem ipsum dolor sit amet consectetur, adipisicing elit.<br>Lorem ipsum dolor sit amet consectetur, adipisicing elit.   
+                          </p>
+                      </div>
+                    </div>
+                  <div class="footer-container-mobile">
+                      <div class="container-child-mobile">
+                        <div class="footer-title-mobile">Creative commons DRC</div>
+                        <div class="footer-content-mobile">
+                           <p class="footertext-mobile">
+                             2022 creative common DR.Congo 
+                           </p>
+                        </div>
+                      </div>
+                  </div>
```

Comment:
- [ ] you have two index file. `index.html` and `index2.html`. you can remove one to make your code more clean.